recipes/cascade/run_test.bat

```diff
@@ -0,0 +1 @@
+python -c "from cascade import test; test.run_test_suite()"
```

Comment:
I think you can drop both the .bat and .sh test file for a:

```
test:
   command:
     - test_line from above
```

in the yaml file.